src/libstd/fs/mod.rs

```diff
@@ -128,6 +128,18 @@ impl File {
     ///
     /// This function will return an error if `path` does not already exist.
     /// Other errors may also be returned according to `OpenOptions::open`.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::io::prelude::*;
```

Comment:
I believe the prelude can be omitted for for the examples that don't actually do any reading/writing